torch/distributed/distributed_c10d.py

```diff
@@ -1714,6 +1714,10 @@ def broadcast_object_list(object_list, src=0, group=None):
         src (int): Source rank from which to broadcast ``object_list``.
         group: (ProcessGroup, optional): The process group to work on. If None,
             the default process group will be used. Default is ``None``.
+        dist_device (``torch.device``, optional): If not None, then intermediate
```

Comment:
Curious, any reason for choosing the name `dist_device` instead of `device` or `map_location` or other names? Is the `dist_`  prefix here stands for distributed? If so, why this is an "distributed device"?

Reply:
+1, maybe just call it `device`?

Reply:
mainly b/c dist_device was mentioned here https://github.com/pytorch/pytorch/issues/60062 :) 
changed it to device.